src/Superpower/Display/Presentation.cs

```diff
@@ -69,26 +70,45 @@ public static string FormatAppearance<TKind>(TKind kind, string value)
 
             return $"{FormatKind(kind)} {clipped}";
         }
-
         public static string FormatLiteral(char literal)
         {
             switch (literal)
             {
-                case '\r':
-                    return "carriage return";
-                case '\n':
-                    return "line feed";
-                case '\t':
-                    return "tab";
-                case '\0':
-                    return "NUL";
+     
```

Comment:
Extra leading newline here?

Reply:
fixed along with the addition of  control codes.